docs/PersonAPI.md

```diff
@@ -129,7 +129,24 @@ Retrieve specific user profiles:
 - `/v2/user/primary_email/<string:primary_email>`
 - `/v2/user/primary_username/<string:primary_username>`
 
-Retrieve lists of users:
+Retrieve paginated list of all user profiles:
+- `/v2/users/`
+
+In this case you will receive a JSON document such as :
+```
+{
+  "Items": [profile_1, profile_2, ...],
+  "nextPage": None
+}
+```
+where `nextPage` will be set to a `user_id` if there are more pages to fetch.
```

Comment:
nextPage actually is an object:
```json
{ "id": "SOME_USER_ID" }
```
I fell for this and it drove me crazy to figure it out.

Reply:
ops didnt submit "review", i though i commented here
after some testing i seem to definitely be getting either none either a single user_id, not a structure

see the code also https://github.com/mozilla-iam/cis/pull/411/files#diff-3247471fbaab5e22352bc0bfc86e2063R244 this definitely gets all pages